docs/tutorials/plotting/pairplot_with_gmm.ipynb

```diff
@@ -0,0 +1,130 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Pairplot: Visualizing High Dimensional Data\n",
+    "\n",
+    "This example provides how to visualize high dimensional data using the pairplot."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import graspologic\n",
+    "\n",
+    "import numpy as np\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Simulate a binary graph using stochastic block model\n",
+    "The 3-block model is defined as below:\n",
+    "\n",
+    "\\begin{align*}\n",
+    "n &= [50, 50, 50]\\\\\n",
+    "P &= \n",
+    "\\begin{bmatrix}0.5 & 0.1 & 0.05 \\\\\n",
+    "0.1 & 0.4 & 0.15 \\\\\n",
+    "0.05 & 0.15 & 0.3\n",
+    "\\end{bmatrix}\n",
+    "\\end{align*}\n",
+    "\n",
+    "Thus, the first 50 vertices belong to block 1, the second 50 vertices belong to block 2, and the last 50 vertices belong to block 3."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from graspologic.simulations import sbm\n",
+    "\n",
+    "n_communities = [50, 50, 50]\n",
+    "p = [[0.5, 0.1, 0.05], \n",
+    "     [0.1, 0.4, 0.15], \n",
+    "     [0.05, 0.15, 0.3],]\n",
+    "\n",
+    "np.random.seed(2)\n",
+    "A = sbm(n_communities, p)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Embed using adjacency spectral embedding to obtain lower dimensional representation of the graph\n",
+    "\n",
+    "The embedding dimension is automatically chosen. It should embed to 3 dimensions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from graspologic.embed import AdjacencySpectralEmbed\n",
+    "\n",
+    "ase = AdjacencySpectralEmbed()\n",
+    "X = ase.fit_transform(A)\n",
+    "\n",
+    "print(X.shape)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Use pairplot_with_gmm to plot the embedded data with ellipses\n",
+    "\n",
+    "First we generate labels that correspond to blocks. We pass the labels along with the data for to create a pairplot with the ellipses estimated by the GMM over each compnonent of the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from graspologic.plot import pairplot_with_gmm\n",
+    "from sklearn.mixture import GaussianMixture\n",
+    "\n",
+    "gmm = GaussianMixture(n_components=3, covariance_type=\"full\").fit(X)\n",
+    "\n",
+    "labels = ['Block 1'] * 50 + ['Block 2'] * 50 + ['Block 3'] * 50\n",
+    "\n",
+    "plot = pairplot_with_gmm(X, gmm, labels)"
```

Comment:
@loftusa in order to prevent permutation of color I pretty much just searched for the nearest means in the predicted clusters. once I make another push ill comment on it for you to see. But it fixes the problem rather easily and efficiently without much verbose code.

Reply:
was this resolved? if so, link commit in this convo

Reply:
Well it was resolved in the sense because i am no longer need to worry about this https://github.com/microsoft/graspologic/pull/541/commits/ef687dae45d4d016851023ae356b6d5b4a1654aa